content/src/storage/repositories/DeploymentsRepository.ts

```diff
@@ -71,14 +78,52 @@ export class DeploymentsRepository {
       offset
     }
 
+    if (lastEntityId) {
+      values.lastEntityId = lastEntityId
+    }
+
+    /**  The lastEntityId is a field that we only want to compare with if for pagination.
+     * If the filter specifies a timestamp value that it's repeated among many deployments,
+     * then to know where the page should start we will use the lastEntityId.
+     */
+    const pageBorder: string =
+      (order == SortingOrder.ASCENDING ? 'from' : 'to') +
+      (timestampField == SortingField.ENTITY_TIMESTAMP ? 'EntityTimestamp' : 'localTimestamp')
```

Comment:
Shouldn't `localTimestamp` be `LocalTimestamp`? If it should, then we don't have enough tests.

Note: I added a comment about it in the test file